### DISCUSSION TITLE
如何添加ICP备案号？

### DISCUSSION THREAD

**Discussion by JamesZhou233** [AlistGo/alist#5566] [Q&A]
整个网盘使用了Alist进行访问，并且为其申请了域名。国家要求备案之后添加ICP备案号。请问如何添加？

**Comment by BlueSkyXN** [comment]
建议不添加，也不要用WWW，因为可能不符合网安政策

**Reply by JamesZhou233** [reply]
目前域名下的正式网站还没有做完，比较成熟的也就是这个alist服务器了。不加怕不符合ICP备案之后悬挂ICP备案号的要求😅

**Reply by momoshuo** [reply]
新建个首页，写几个字就好了，千万不要把网盘挂ICP

**Comment by goldsoft8888** [comment]
域名NS 解析到国外，然后DNS解析到家庭宽带，这样不受那些乱七八糟的管制
